backend/webhook/publisher/src/main/java/co/airy/core/webhook/publisher/Mapper.java

```diff
@@ -19,16 +21,23 @@
     }
 
     public WebhookBody fromMessage(Message message) throws Exception {
-        contentMapper.render(message);
```

Comment:
Gee someone completely forgot about this (yes it was me)
![tenor](https://user-images.githubusercontent.com/4403838/100769964-01826f80-33fd-11eb-9e18-49458c9cc61d.gif)